Tutorials/Members-Registration-And-Logins/index-v10.md

```diff
@@ -0,0 +1,360 @@
+needsV9Update: "true"
+---
+
+# Member registration and login 
+
+Using tools available on a fresh install of Umbraco CMS, you are able to create a frontend-based registration and login functions, and restrict access to specific areas of your site based on this system.
+
+By the end of this tutorial you will know how to implement a basic register/login functionality on your website, hide pages from non-logged in members, and assign newly registered members to specific member groups.
+
+
+## Create the necessary macros and partial files
+
+Navigate to the Settings dashboard in the Backoffice, and under Templating section, expand Partial View Macro Files folder and create a new partial view.
+
+![Partial create](images/v8-01-Partial-Macro-Create.png)
+
+Make sure to create this Macro view from available snippets.
+
+![Partial macro snippets](images/v8-02-Partial-Macro-Snippets.png)
+
+Start with the Register Member snippet.
+
+![Partial macro register](images/v8-03-Partial-Macro-Register.png)
+
+Make sure to fill out the name of the partial view (you can call it "Register", for example).
+Afterwards, using the same approach, create a Macro Partial View using the Login snippet and name it "Login".
+
+With that done, navigate to Partial Views folder above Partial View Macro Files, and create a New partial view from snippet.
+This view should be using the Login Status snippet.
+
+![Partial snippets](images/v8-04-Partial-Snippets.png)
+
+These three partials already contain all the necessary code to make the register, login and logout functionality work.
+
+Since the Register and Login were created as macro partials, you will have corresponding Macros with the same names available under the Macros folder above the Templating section.
+Make sure to expand the Macros view, and in Editor Settings enable Login and Register macros to be used in Richtext Editor and the grid.
+
+![Macro settings](images/v8-05-Macros-Settings.png)
+
+## Include the Macros and Partials in Content and Templates
+
+Halfway there! Next, navigate to your Content section. Since we created Register and Login partials as macro partials, we can paste them in a Richtext Editor.
+Create a new page that is using this editor, give it a name (e.g. Login/Signup), and in any RTE or Grid layouts you have on the page, insert a Macro.
+
+![RTE macro](images/v8-06-RTE-Macro.png)
+
+Choose the Macro you created. Depending on how you would like to structure your signup/login pages, you can include one of the macros or place both of them next to each other.
+
+![RTE macro insert](images/v8-07-RTE-Macro-Insert.png)
+
+Save and publish the new page. Now the Register/Login functionality should be rendered on the new page you made.
+
+![Login page frontend](images/v8-08-Login-Page-Frontend.png)
+
+With the above macros in place, you can now use the page to register new Members.
+Every registered person will show up in the Members section in the backoffice.
```

Comment:
```suggestion
Halfway there! Next, navigate to your Content section. Since we created Register and Login partials as macro partials, we can paste them into a Richtext Editor.
Create a new page that is using this editor, give it a name (e.g. Login/Signup), and in any RTE or Grid layouts you have on the page, insert a Macro.

![RTE macro](images/v8-06-RTE-Macro.png)

Choose the Macro you created. Depending on how you would like to structure your signup/login pages, you can include one of the macros or place both of them next to each other.

![RTE macro insert](images/v8-07-RTE-Macro-Insert.png)

Save and publish the new page. Now the Register/Login functionality should be rendered on the new page you made.

![Login page frontend](images/v8-08-Login-Page-Frontend.png)

With the above macros in place, you can now use the page to register new Members.
Every registered person will show up in the Members section in the backoffice.
```